MultiTable/src/RealtimeChangeHandler.js

```diff
@@ -74,9 +74,13 @@ function onSubjectAdd(data, subject) {
     if (!subjectGroup) {
       const parentPath = subject.parentAbsolutePath;
       const grandparentGroup = data.getParentGroupForAbsolutePath(parentPath);
-      if (!grandparentGroup) return;
+      if (!grandparentGroup) {
```

Comment:
FYI it's OK to leave these as single line per style guide

Reply:
I think it's more readable as a single line for simple returns like this.

Reply:
Most of the if blocks in this repo and other refocus related repos have brackets and I want to make sure it is consistent everywhere.